corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -63,6 +63,8 @@
 
 
 logger = logging.getLogger(__name__)
+log = logging.getLogger(__name__)
+
```

Comment:
This is not necessary. In this module you can use the one that's already created on the line above (`logger`). Sorry for the naming confusion.